doc/nrf/config_and_build/sysbuild_migration.rst

```diff
@@ -0,0 +1,579 @@
+| :kconfig:option:`SB_CONFIG_DFU_ZIP_NET`                     | Include network-core image update in zip archive                           |
++-------------------------------------------------------------+----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_DFU_ZIP_WIFI_FW_PATCH`           | Include nRF700x firmware patch update in zip archive                       |
++-------------------------------------------------------------+----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_DFU_ZIP_BLUETOOTH_MESH_METADATA` | Include Bluetooth mesh metadata in zip archive                             |
++-------------------------------------------------------------+----------------------------------------------------------------------------+
+
+.. _child_parent_to_sysbuild_migration_partition_manager:
+
+Partition manager
+#################
+
++-----------------------------------------------------------------+----------------------------------------------------------------------------+
+| Kconfig option                                                  | Description                                                                |
++=================================================================+============================================================================+
+| :kconfig:option:`SB_CONFIG_PARTITION_MANAGER`                   | Enables partition manager support                                          |
++-----------------------------------------------------------------+----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_PM_MCUBOOT_PAD`                      | MCUboot image header padding                                               |
++-----------------------------------------------------------------+----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY` | Places the secondary MCUboot update partition in external flash            |
++-----------------------------------------------------------------+----------------------------------------------------------------------------+
+| :kconfig:option:`SB_CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK`   | Will force override the external flash driver check                        |
++-----------------------------------------------------------------+----------------------------------------------------------------------------+
+
+.. _child_parent_to_sysbuild_migration_filename_changes:
+
+Filename changes
+################
+
+Some output file names have changed from child/parent image or have changed the directory that they are created in, this is because sysbuild properly namespaces images in a project. The changes to final output files (ignoring artefacts and intermediary files) are as following:
```

Comment:
```suggestion
Some output file names have changed from child/parent image configurations or have changed the directory where they are created.
This is because Sysbuild properly namespaces images in a project.
The changes to final output files (ignoring artifacts and intermediary files) are as follows:
```